aiormq/base.py

```diff
@@ -40,34 +40,48 @@ def __repr__(self):
 
 
 class FutureStore:
-    __slots__ = "futures", "loop", "parent"
+    __slots__ = "futures", "important_futures", "loop", "parent"
 
```

Comment:
Maybe introduce self.future_sets: List[int, Set[TaskWrapper]] instead of new "important_futures" and "futures". int -> priority level. (Smaller is more important)
And reject all tasks by priority >= param